appcues/src/main/java/com/appcues/analytics/AutoPropertyDecorator.kt

```diff
@@ -132,6 +133,9 @@ internal class AutoPropertyDecorator(
             AnalyticsEvent.DeviceUpdated.eventName -> {
                 attributes[DEVICE_PROPERTY] = deviceProperties
             }
+            AnalyticsEvent.PushOpened.eventName -> {
+                attributes[DEVICE_ID] = storage.deviceId
```

Comment:
when decorating PushOpened we include the DEVICE_ID (cannot be accessed when we receive the message or when we handle the deeplink)